hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/resolver/NamenodeStatusReport.java

```diff
@@ -382,7 +384,8 @@ public void setNamesystemInfo(long available, long total,
     this.numOfBlocksPendingDeletion = numBlocksPendingDeletion;
     this.numOfFiles = numFiles;
     this.statsValid = true;
-    this.providedSpace = providedSpace;
+    this.providedSpace = providedStorageSpace;
```

Comment:
if this is just formatting change, please avoid this.

Reply:
This is to fix [checkstyles](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4009/5/artifact/out/results-checkstyle-root.txt). I don't know if I need to update the checkstyles of the old code.